scientific/spectroscopy/specpr.ksy

```diff
@@ -0,0 +1,254 @@
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
```

Comment:
You've beaten your previous record here ;)

Your file has 50K of binary zeroes appended to it instead of a trailing newline. Plus broken line endings (somehow your git haven't converted CRLF -> LF on checkin).